asdf/_asdf.py

```diff
@@ -117,16 +117,15 @@ def __init__(
             as-is.
 
         copy_arrays : bool, optional
+            Deprecated; use ``memmap`` instead.
             When `False`, when reading files, attempt to memmap underlying data
             arrays when possible.
 
         memmap : bool, optional
             When `True`, when reading files, attempt to memmap underlying data
             arrays when possible. When set, this argument will override
-            ``copy_arrays``.  When not set, the ``copy_arrays`` will determine
-            if arrays are memory mapped or copied. ``copy_arrays`` will be
-            deprecated and the default will change in an upcoming asdf version
-            which by default will not memory map arrays.
+            ``copy_arrays``. The default will change to ``False`` in an upcoming
+            ASDF version.
```

Comment:
```suggestion
            ASDF version. At the moment the default is ``True``.
```
I think since both copy_arrays and memmap are `NotSet` in the arguments some description of the default state here would be helpful. This is only temporary since in asdf 4.0 we can remove the `NotSet` and switch it to the default.